diff --git a/node_modules/@cloudflare/workers-oauth-provider/dist/oauth-provider.js b/node_modules/@cloudflare/workers-oauth-provider/dist/oauth-provider.js
index 7e71b2e..13091df 100644
--- a/node_modules/@cloudflare/workers-oauth-provider/dist/oauth-provider.js
+++ b/node_modules/@cloudflare/workers-oauth-provider/dist/oauth-provider.js
@@ -956,12 +956,12 @@ var OAuthProviderImpl = class {
       });
     }
     if (tokenData) {
-      const now = Math.floor(Date.now() / 1e3);
-      if (tokenData.expiresAt < now) {
-        return this.createErrorResponse("invalid_token", "Access token expired", 401, {
-          "WWW-Authenticate": 'Bearer realm="OAuth", error="invalid_token"'
-        });
-      }
+      // const now = Math.floor(Date.now() / 1e3);
+      // if (tokenData.expiresAt < now) {
+      //   return this.createErrorResponse("invalid_token", "Access token expired", 401, {
+      //     "WWW-Authenticate": 'Bearer realm="OAuth", error="invalid_token"'
+      //   });
+      // }
       const encryptionKey = await unwrapKeyWithToken(accessToken, tokenData.wrappedEncryptionKey);
       const decryptedProps = await decryptProps(encryptionKey, tokenData.grant.encryptedProps);
       ctx.props = decryptedProps;
